_gsocblogs/2022/blog_Acts_TirthankarMazumder.md

```diff
@@ -117,14 +117,60 @@ Apart from my work on Fast5×5, I also fired off a [small PR](https://github.com
+
+<center><a href="https://user-images.githubusercontent.com/63574588/203063987-10bea51c-0fb0-4315-bc6d-d9953f7c2ffb.png"><img src="https://user-images.githubusercontent.com/63574588/203063987-10bea51c-0fb0-4315-bc6d-d9953f7c2ffb.png" width="800" height="450"></a></center>
+<center><caption>Benchmark plot for the similarity transform with <code>double</code> matrices.</caption></center>
+
+<center><a href="https://user-images.githubusercontent.com/63574588/203063866-3120d95f-d17e-4847-9e5a-793b57bd2073.png"><img src="https://user-images.githubusercontent.com/63574588/203063866-3120d95f-d17e-4847-9e5a-793b57bd2073.png" width="800" height="450"></a></center>
+<center><caption>Benchmark plot for the free coordinates to bound coordinates transform with <code>float</code> matrices.</caption></center>
+
+<center><a href="https://user-images.githubusercontent.com/63574588/203063885-88a3f2f2-3550-4296-ad30-ca9d98551ee5.png"><img src="https://user-images.githubusercontent.com/63574588/203063885-88a3f2f2-3550-4296-ad30-ca9d98551ee5.png" width="800" height="450"></a></center>
+<center><caption>Benchmark plot for the free coordinates to bound coordinates transform with <code>double</code> matrices.</caption></center>
+
+<center><a href="https://user-images.githubusercontent.com/63574588/203063883-e7d80d2a-d265-4ee6-9195-0ef9085f1d18.png"><img src="https://user-images.githubusercontent.com/63574588/203063883-e7d80d2a-d265-4ee6-9195-0ef9085f1d18.png" width="800" height="450"></a></center>
+<center><caption>Benchmark plot for the bound coordinates to free coordinates transform with <code>float</code> matrices.</caption></center>
+
+<center><a href="https://user-images.githubusercontent.com/63574588/203063885-88a3f2f2-3550-4296-ad30-ca9d98551ee5.png"><img src="https://user-images.githubusercontent.com/63574588/203063885-88a3f2f2-3550-4296-ad30-ca9d98551ee5.png" width="800" height="450"></a></center>
+<center><caption>Benchmark plot for the bound coordinates to free coordinates transform with <code>double</code> matrices.</caption></center>
+
+It is clear from these plots that Fastor is consistently near the top (i.e. either the fastest or the second-fastest in all of these tasks).
+
+## Integrating Fastor into `algebra-plugins`
+
+I first had to define the math functions that live in the [`math` subdirectory](https://github.com/wermos/algebra-plugins/tree/528327c51d46cf198fe79fd83856c47b29d6d2b1/math/fastor) in terms of `Fastor` primitives and functions. After that, I had to declare the standard litany of types in the [`storage` subdirectory](https://github.com/wermos/algebra-plugins/tree/528327c51d46cf198fe79fd83856c47b29d6d2b1/storage/fastor). Then, I had to define the [`fastor_fastor` frontend](https://github.com/wermos/algebra-plugins/tree/528327c51d46cf198fe79fd83856c47b29d6d2b1/frontend/fastor_fastor) so that the backend is actually usable and the tests can be written.
```

Comment:
This seems a bit incoherent to me, tbh. I had hoped for a final wrap-up of the project, along the lines of a bit of context of the project, a short reflection on the benchmarks (Eigen vs blaze vs fastor etc). Maybe a short introduction to fastor? OK, there is a more detailed overview about algebra-plugins in your blog...

Reply:
Just telling the overarching story of the project